frontend/www/js/omegaup/components/contest/AddProblem.vue

```diff
@@ -248,11 +290,13 @@ export default class AddProblem extends Vue {
     this.$emit('runs-diff', this.alias, versions, selectedCommit);
   }
 
+  get isUpdate(): boolean {
+    return this.problems.some((problem) => problem.alias === this.alias);
```

Comment:
```suggestion
    return !!this.problemMapping[this.alias];
```

ya tenemos un mappeo, así que podemos usarlo! y así tenemos muchas mejores garantías de que `onGetVersions` no va a obtener nada no-definido (buena idea la de crear el mappeo!)

Reply:
Listo